### ISSUE TITLE
Target line is enormous when control is scaled up

### ISSUE THREAD

**Issue by danielcaron** [okviz/BulletChart#22]
I love this visualization and I use it often.  When scaling the BulletChart size up, the thickness of the Target line becomes very, very thick.  The impact is that the line is so thick you can no longer determine the value to a meaningful degree of accuracy.  See attached.  I think the solution might be to allow us to control the thickness...or, don't scale the line thickness at all.

![bulletcontrolscreenshot](https://user-images.githubusercontent.com/14171516/44794328-4829f500-ab76-11e8-8a85-0ff09a72c2a1.png)

**Comment by danieleperilli** [comment]
Yes, it is an addition we will include in one of the next releases.
